src/tokens.js

```diff
@@ -0,0 +1,39 @@
+import {Bool, Ignoring, On, GroupLeft, GroupRight, Offset, Avg, Bottomk, Count, CountValues, Group, Max, Min, Quantile, Stddev, Stdvar, Sum, Topk, By, Without, And, Or, Unless} from './parser.terms.js';
+import { aggregateOpTerms } from './promql.terms.js';
```

Comment:
Are you using this import somewhere in the file ? I can't find where

Reply:
Ah I was initially, but then didn't end up using it. Fixed, thanks.